internal/btf/btf.go

```diff
@@ -193,67 +194,79 @@ func loadSpecFromELF(file *internal.SafeELFFile) (*Spec, error) {
 // transforms them to be indexed by function. Retrieves function names from
 // the BTF spec.
 func (spec *Spec) splitExtInfos(info *extInfo) error {
-
 	ofi := make(map[string]FuncInfo)
 	oli := make(map[string]LineInfos)
 	ocr := make(map[string]CoreRelos)
 
 	for secName, secFuncs := range info.funcInfos {
```

Comment:
We should probably rename `secFuncs` to `secFuncInfos` with the addition of `funcs` below. This function didn't become easier to grok.